Mathlib/Analysis/NormedSpace/PiLp.lean

```diff
@@ -81,6 +81,9 @@ abbrev PiLp (p : ℝ≥0∞) {ι : Type*} (α : ι → Type*) : Type _ :=
 instance (p : ℝ≥0∞) {ι : Type*} (α : ι → Type*) [∀ i, Inhabited (α i)] : Inhabited (PiLp p α) :=
   ⟨fun _ => default⟩
 
+instance (p : ℝ≥0∞) {ι : Type*} [Inhabited ι] (α : ι → Type*) [∀ (i : ι), Nonempty (α i)]
+    [Nontrivial (α default)] : Nontrivial (PiLp p α) := Pi.nontrivial
```

Comment:
I think this should be an instance in `WithLp.lean`, that says "if `V` is nontrivial then so is `WithLp p V`"

Reply:
Yes, that makes sense. Here it comes.